### PR TITLE
documentation utilisateur / user doc

### PR DESCRIPTION
**Pull request by Saimyx**
J'ai essayé d'écrire un peu de documentation utilisateur vu qu'elle était très légère.
Est-ce que ça va comme ça ?

I tried to write in doc user. It's good ?

### CODE REVIEW

File: docs/en/users/04_Subscriptions.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
 # Adding a feed
 
-**TODO**
+ 1. To add a feed, copy the URL to RSS or Atom file's of the content wanted (for exemple, the framablog RSS URL is `https://framablog.org/feed/`).
```
Comment (Alkarex) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 1. To add a feed, copy the URL to RSS or Atom file's of the content wanted (for instance, the Framablog RSS URL is `https://framablog.org/feed/`).
```

**Comment by Alkarex** [comment]
Merci :-)

File: docs/en/users/04_Subscriptions.md (hunk 1 above)
Comment (Frenzie) [reply docs/en/users/04_Subscriptions.md]: For example is also fine, just for ex*e*mple isn't. ^_^

File: docs/en/users/04_Subscriptions.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+ 2. On FreshRSS, clic to “Subscriptions management”.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 2. On FreshRSS, click “Subscriptions management”.
```

Comment (Frenzie) [reply docs/en/users/04_Subscriptions.md]: (That should be subscription management really, but that's unrelated to this PR.)

File: docs/en/users/04_Subscriptions.md [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+## Export
+
+ 1. To export your list of fedds, go to “Subscriptions management”.
+ 2. Clic on "Import / export"
+ 3. You can select for your export :
+    1. the list of feeds
+    2. the labelled articles
+    3. the favurites articles
+    4. and finally, you can to select feeds you want to export (by default, all feeds are selected)
+ 4. Clic on "export".
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 4. Click on "export".
```

File: docs/en/users/04_Subscriptions.md [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
+ 5. Save your file.
+ 
+ ## Import
+ 
+  1. Go to the page "Import / export".
+  2. Clic on "Browse" and select your file on your computer.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
  2. Click on "Browse" and select your file on your computer.
```

File: docs/en/users/04_Subscriptions.md [hunk 5]
```diff
@@ -1,11 +1,31 @@
...
+  3. Clic on "Import"
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
  3. Click on "Import"
```

File: docs/en/users/04_Subscriptions.md [hunk 6]
```diff
@@ -16,4 +36,13 @@ Bookmarklets are little scripts that you can execute to perform various tasks. F
 
 # Feed management
 
-**TODO**
+You can manage your feeds with differents category. You can do it when addind a feed (top) and when you want. A feed can't be in many category.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
You can manage your feeds within different categories. You can select a category while adding a feed (top) or whenever you want. A feed can't be in multiple categories.
```

File: docs/en/users/04_Subscriptions.md [hunk 7]
```diff
@@ -16,4 +36,13 @@ Bookmarklets are little scripts that you can execute to perform various tasks. F
...
+
+ 1. Open "Subscriptions management".
+ 2. You can add a category :
+    1. Type the name of your category in "New category"
+    2. Clic on "Submit"
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
    2. Click on "Submit"
```

File: docs/en/users/04_Subscriptions.md [hunk 8]
```diff
@@ -16,4 +36,13 @@ Bookmarklets are little scripts that you can execute to perform various tasks. F
...
+ 3. You can move a feed on the category you want
+ 4. (optional) : If you want a feed display on his category and not in main timeline, clic on the "wheel" of your feed.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 4. (optional) : If you want a feed display on his category and not in main timeline, click on the "wheel" of your feed.
```

File: docs/fr/users/04_Subscriptions.md [hunk 9]
```diff
@@ -1,23 +1,53 @@
 # Ajouter un flux
 
-**À FAIRE**
+ 1. Pour ajouter un flux, copiez le lien vers le fichier RSS ou Atom qui vous intéresse (par exemple, le lien RSS du framablog est `https://framablog.org/feed/`).
```
Comment (Frenzie) [inline docs/fr/users/04_Subscriptions.md]:
```suggestion
 1. Pour ajouter un flux, copiez le lien vers le fichier RSS ou Atom qui vous intéresse (par exemple, le lien RSS du Framablog est `https://framablog.org/feed/`).
```

**Review by Frenzie — COMMENTED** [review]
The structure seems decent enough, added some basic spelling & grammar corrections.

File: docs/en/users/04_Subscriptions.md (hunk 1 above)
Comment (Saimyx) [reply docs/en/users/04_Subscriptions.md]: Oh yes, shame on me...
Thank you !

**Comment by Saimyx** [comment]
Thank you very much for corrections, I am not very good in english...

**Comment by Alkarex** [comment]
Thanks @Saimyx 
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md

**Pull request merged**